client/src/client.ts

```diff
@@ -0,0 +1,65 @@
+import * as path from 'path';
+import { execFile } from 'child_process';
+import { workspace, ExtensionContext } from 'vscode';
+import { waitForSocket } from 'socket-retry-connect';
+import { Socket } from 'net';
+
+import {
+    LanguageClient,
+    LanguageClientOptions,
+    StreamInfo
+} from 'vscode-languageclient';
+
+let client: LanguageClient;
+
+function startServer(context: ExtensionContext) {
+    let serverPort = 10000;
+    let serverPath = context.asAbsolutePath(
+        path.join('erlang_ls', '_build', 'default', 'rel', 'erlang_ls', 'bin', 'erlang_ls')
```

Comment:
This seems outdated. This should point to `erlang_ls/_build/default/bin/erlang_ls`. Also, the server now supports stdio as well, so it's up to the client to decide which transport to use. See the README for the escript arguments.

Reply:
Updated to the shorter path.